### PR TITLE
Tegu-class Communications Freighter

### PR DESCRIPTION
**Pull request by goober3**
## About The Pull Request
This PR adds a brand new ship to Nanotrasen's post-rework roster.

<details><summary>Images</summary>

68747470733a2f2f616666656374656461726330372e626c6f622e636f72652e77696e646f77732e6e65742f6d6462322f696d616765732f3336373235323037352f33333231373933373937312f612f5f6d6170735f73687574746c65735f6e616e6f74726173656e5f

**images below are outdated**
image

image

</details>

The Tegu-class is an engineering freighter equipped with a full telecommunications setup and a subshuttle. It is built to not only facilitate communications between Nanotrasen-aligned vessels in the sector, but also foster several smaller networks for other vessels with a small array of relays. The ship also comes equipped with a small cargo department, an engineering department, a small communications room, and a supply office for the QM.

This PR also adds a new design disk for printing telecommunications parts. 

To-do:
- [x] Ship description.
- [ ] In-game testing.
- [x] Add security role.
- [ ] Mining facilities.

## Why It's Good For The Game
more ship

## Changelog

🆑goober3, Cloudbreak
add: The year is 2070. The Tegu-class Communications Freighter has just been fullmerged.
add: A new subshuttle, the Skink-class Cargo Runner.
/🆑

### CODE REVIEW

**Comment by EEASAS** [comment]
okkk.... but isnt there like.... new ship restrictions.... like you cant make new nanotrasen/independent/syndicate/srm/solgov ships......

**Comment by meemofcourse** [comment]
do they know chat

**Comment by goober3** [comment]
> okkk.... but isnt there like.... new ship restrictions.... like you cant make new nanotrasen/independent/syndicate/srm/solgov ships......

i'm doing this in defiance of the maptainers because i hate them

**Comment by thgvr** [comment]
> okkk.... but isnt there like.... new ship restrictions.... like you cant make new nanotrasen/independent/syndicate/srm/solgov ships......

This restriction is for contributors outside of the main team.

**Comment by rye-rice** [comment]
sowwy, nwo prwogwess iwn mwonths, rweopwen wwen wrorking on iwt agwian

**Comment by goober3** [comment]
we're so back

**Comment by thgvr** [comment]
I am begging you to finish this

**Comment by goober3** [comment]
> I am begging you to finish this

this is technically already finished. it just needs a review and a description.

a telecomms guide too maybe

**Review by meemofcourse — CHANGES_REQUESTED** [review]
imagen
i do not like that the lightswitch in the officer's lounge controls the lights in the officers' quarters. there are also three light switches in the cargo office
imagen
this restroom needs better lighting
imagen
i believe the pda painter is a centcom utility and it lets you have SUNS pdas so it would be best to remove it
imagen
the only set of tools i could find in this ship are hidden below two welding helmets. consider adding toolboxes and toolbelts to the engineer and engi director lockers - they should not start with them
imagen
maybe have an emergency suit here
imagen
while i'm not an engi player i don't think this is right

maybe a ship like this could use a power monitor. i'd ask a maptainer/other person who is more experienced with telecommunications to test things out but cool ship

**Comment by goober3** [comment]
> i do not like that the lightswitch in the officer's lounge controls the lights in the officers' quarters. there are also three light switches in the cargo office

this was done so i wouldn't have to make more areas and add APCs to the officer's quarters but in hindsight i probably should just do that. also the multiple lightswitches exist for the same reason you'd have two light switches on both ends of a hallway. it's more convenient like that.

> i believe the pda painter is a centcom utility and it lets you have SUNS pdas so it would be best to remove it

i might add a generic sanitised version of the PDA painter later in a seperate PR. for now they can live without.

thak yuo

**Comment by fighterslam** [comment]
The "/turf/open/floor/plasteel/telecomms_floor" (north floor) starts with the normal, default airmix, 101.3 KPA, etc. The circuit floors (south floor) "turf/open/floor/circuit/telecomms" constantly produce a colder airmix, causing these two adjacent turfs to have atmos differences and process once they spawn in, so I think this might be a small performance concern. It was something I had to deal with on the Ranger too - you probably either want to set the circuit floors to normal circuit floors so there's no atmos differences, or make a new telecomms_floor type that has the same initial airmix. 
image

**Review by meemofcourse — APPROVED** [review]
this ship rocks dude

**Comment by rye-rice** [comment]
@shiptest-ss13/maptainers

File: _maps/configs/nanotrasen_tegu.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
```
Comment (Sadhorizon) [inline _maps/configs/nanotrasen_tegu.json:1]: Add `"faction": "/datum/faction/nt"`, somewhere, or it's gonna show as indie on the spawning menu.

**Comment by thgvr** [comment]
@shiptest-ss13/maptainers ancient pr, please decide if its being merged or not

**Comment by goober3** [comment]
> @shiptest-ss13/maptainers ancient pr, please decide if its being merged or not

this still has yet to be tested substantially in-game actually. some day...

**Comment by DorfGuy1** [comment]
Where's the guide?
 TO the comms

**Comment by tzui5020** [comment]
It needs a chair or a few railings in the Cargobay please. I beg you

**Comment by thgvr** [comment]
purgatory

**Review by meemofcourse — APPROVED** [review]
BWAAH SCARY TELECOMMS

**Pull request merged**